javascript/javascript1/week4/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="script.js"></script>
+</body>
+</html>
```

Comment:
Good job.There are also string methods like 'startsWith' and 'endsWith' just informing for your reference

Reply:
Got it. Thanks :)